db/music_pg.sql

```diff
@@ -0,0 +1,373 @@
+CREATE TABLE "submission" (
+  "submission_id" uuid PRIMARY KEY DEFAULT (gen_random_uuid()),
+  "participant_id" int NOT NULL,
+  "bracket_id" int NOT NULL,
+  "song_id" int NOT NULL,
+  "order" int NOT NULL,
+  "date_submitted" timestamp NOT NULL DEFAULT (now()),
+  "active" bool_int NOT NULL,
+  "is_dupe" bool_int NOT NULL,
+  "title" text NOT NULL,
+  "version" text,
+  "artist" text,
+  "collaborators" text,
+  "composer" text,
+  "link" text NOT NULL,
+  "genre" text,
+  "emoji" text,
+  "comments" text
+);
+
+CREATE TABLE "submission_history" (
+  "submission_history_id" uuid PRIMARY KEY DEFAULT (gen_random_uuid()),
+  "date_update" timestamp NOT NULL,
+  "note_update" text NOT NULL,
+  "submission_id" int NOT NULL,
+  "participant_id" int NOT NULL,
+  "bracket_id" int NOT NULL,
+  "song_id" int NOT NULL,
+  "order" int NOT NULL,
+  "date_submitted" timestamp NOT NULL DEFAULT (now()),
+  "active" bool_int NOT NULL,
+  "is_dupe" bool_int NOT NULL,
+  "title" text NOT NULL,
+  "version" text,
+  "artist" text,
+  "collaborators" text,
+  "composer" text,
+  "link" text NOT NULL,
+  "genre" text,
+  "emoji" text,
+  "comments" text
+);
+
+CREATE TABLE "link_relation" (
+  "link_realtion_id" uuid PRIMARY KEY DEFAULT (gen_random_uuid()),
+  "relation_id" int NOT NULL,
+  "link_id" int NOT NULL,
+  "type_link" text NOT NULL
```

Comment:
If I'm reading this right, I think this should actually be `relation_type`, since it's the relation not the link that is from some set of tables that needs to be specified here?

Reply:
Yes, I think that's right.